erpnext/controllers/accounts_controller.py

```diff
@@ -1060,6 +1060,9 @@ def update_invoice_status():
 	frappe.db.sql(""" update `tabPurchase Invoice` set status = 'Overdue'
 		where due_date < CURDATE() and docstatus = 1 and outstanding_amount > 0""")
 
+	frappe.db.sql(""" update `tabSales Order` set status = 'Overdue'
```

Comment:
Don't set overdue status from background job, it will overwrite "To Deliver" and "To Bill" status, which is more important.